include/exadg/acoustic_conservation_equations/spatial_discretization/operators/weak_boundary_conditions.h

```diff
@@ -65,27 +66,45 @@ template<int dim, typename Number>
 inline DEAL_II_ALWAYS_INLINE //
   dealii::Tensor<1, dim, dealii::VectorizedArray<Number>>
   calculate_exterior_value_velocity(unsigned int const                       q,
-                                    FaceIntegrator<dim, dim, Number> const & integrator_m,
+                                    FaceIntegrator<dim, dim, Number> const & velocity_m,
+                                    FaceIntegrator<dim, 1, Number> const &   pressure_m,
+                                    double const                             c,
```

Comment:
We should use here `speed_of_sound` as is used below for the `BoundaryFaceIntegrator`.

Reply:
[4b4c61d](https://github.com/exadg/exadg/pull/629/commits/4b4c61dfcfc1ab1fe9137668b6ec6d09ab57e672)